chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -356,7 +369,18 @@ void BraveRenderViewContextMenu::InitMenu() {
   RenderViewContextMenu_Chromium::InitMenu();
 
   int index = -1;
+
+  index = menu_model_.GetIndexOfCommandId(
+      IDC_CONTENT_CONTEXT_PASTE_AND_MATCH_STYLE);
+  if (index != -1) {
+    menu_model_.InsertItemWithStringIdAt(
+          index + 1,
+          IDC_CONTENT_CONTEXT_FORCE_PASTE,
+          IDS_CONTENT_CONTEXT_FORCE_PASTE);
+  }
+
 #if BUILDFLAG(ENABLE_TOR)
+  index = -1;
```

Comment:
is it needed at all?

Reply:
needed for https://github.com/brave/brave-core/pull/13178/files/8497a6007ed1f95683343451ccdb8ca60c94dcb6#diff-98e5cd23cdc68113dd56309d734c30615eee2597d8fe699ff51d0637262dfce6R392

Reply:
at first I was thinking it was, but on second look I think it can be removed